api/alter_job.md

```diff
@@ -39,6 +39,9 @@ other useful statistics for deciding what the new schedule should be.
 |`next_start`|`TIMESTAMPTZ`|The next time at which to run the job. The job can be paused by setting this value to `infinity`, and restarted with a value of `now()`|
 |`if_exists`|`BOOLEAN`|Set to `true`to issue a notice instead of an error if the job does not exist. Defaults to false.|
 |`check_config`|`REGPROC`|A function that takes a single argument, the `JSONB` `config` structure. The function is expected to raise an error if the configuration is not valid, and return nothing otherwise. Can be used to validate the configuration when updating a job. Only functions, not procedures, are allowed as values for `check_config`.|
+|`fixed_schedule`|`BOOLEAN`|Set to `TRUE` if want to have fixed schedule job runs.|
```

Comment:
|`fixed_schedule`|`BOOLEAN`| To enable fixed scheduled job runs, set to `TRUE`.  |
|`initial_start`|[TIMESTAMPTZ](https://www.postgresql.org/docs/current/datatype-datetime.html)| Set the time when the `fixed_schedule` job run starts. For example, `19:10:25-07`|
|`timezone`|`TEXT`| Address the 1-hour shift in start time when clocks change from [Daylight Saving Time to Standard Time](https://en.wikipedia.org/wiki/Daylight_saving_time). For example, `example`.|

Can you check the example value for `initial_start` and  add one for `timezone` please.